Libraries/VvVidtk/vvAdaptVidtk.cxx

```diff
@@ -18,12 +18,11 @@
 QList<vvTrackState> vvAdapt(const vidtk::track_state& vs)
 {
   QList<vvTrackState> stateList;
-  std::vector<vidtk::image_object_sptr> objs;
-  if (vs.data_.get(vidtk::tracking_keys::img_objs, objs))
+  vidtk::image_object* obj_sptr = vs.get_image_object();
+  if (obj_sptr != nullptr)
```

Comment:
Please modernize:
```suggestion
  if (auto* const obj = vs.get_image_object())
```

Also, the variable name is wrong:
- It isn't a smart pointer, so should not be named as if it were.
- This code should use camelCase names, not names_with_underscores. (I'm aware some of this code already isn't following that, but let's not exacerbate the problem.)